freshdesk/v2/api.py

```diff
@@ -135,8 +135,12 @@ def list_tickets(self, **kwargs):
             url += "?filter=%s&" % filter_name
         else:
             url += "?"
-        page = kwargs.get("page", 1)
-        per_page = kwargs.get("per_page", 100)
+
+        if "updated_since" in kwargs:
+            url += "updated_since=%s&" % kwargs["updated_since"]
+
+        page = 1 if not "page" in kwargs else kwargs["page"]
```

Comment:
You've brought back in some old code here, we changed this in [83b97d2](https://github.com/sjkingo/python-freshdesk/commit/83b97d27d021f1d994a7c2b46eeed1573bb8a605#diff-33bd36adc6a6677fdf3d3d6fe640eba9R138).

Reply:
This has been fixed now.